packages/components/src/components/tablist/tablist.component.ts

```diff
@@ -166,8 +153,6 @@ class TabList extends Component {
       }
     }
 
-    this.setAriaLabelledByOrAriaLabel();
-
     const resizeObserver = new ResizeObserver(async () => {
       const activeElement = this.tabsContainer?.shadowRoot?.activeElement;
```

Comment:
```suggestion
      const activeElement = document?.activeElement;
```
Accessing the shadowroot's active element returns `null` here and is a bug. This selector should access the document's active element instead